.github/staging/windows_push.yaml

```diff
@@ -50,11 +50,8 @@ jobs:
     - name: Download HAL Plugins and move DLLs into /build

```

Comment:
skipping the review of this file since there is a restriction on windows stage for now . WIP :-)

Reply:
The windows stage is being tested on my github_actions-test branch. Once results are secured, another PR will be sent along.